### ISSUE TITLE
Bimp Creates 2nd Tiff Page - Transparency

### ISSUE THREAD

**Issue by kiselink** [alessandrofrancesconi/gimp-plugin-bimp#378]
Bimp applied to single page tiff file results in 2 page tiff file. Opening tiff with ubuntu document browser shows first image labeled transparency.  

My bimp script is gimp-drawable-levels-stretch plus gimp-drawable-levels. 
Same result on the bimp  "Color corrections" script. 

Tried adapting my bimp script by adding bimp procedures (eg flatten) but no success in removing the transparency (alpha?) layer. An alternative might be to extend the bimp "Change format" script to better support tiff.  Also noticed that many gimp procedures are not accessable via bimp such as those involved with export.  Access to these may or may not have allowed me to remove the transparency.  Noticed bimp operator source code sometimes is mentioning alpha layer - maybe bug there?  

To summarize: Bimp creates a transparency layer in tiff file. An gimp edit of the bimp output displays nonsense although rawtherapee can handle it. imagemagik provides work around.

**Comment by kiselink** [comment]
This turned out to be a gimp 2.10.30 issue. 
It was corrected in 2.10.34.
It expresses itself only in non-interactive mode and gui users do not incur the issue.  
Not a bimp issue!
